core/src/test/resources/OpenApi/envelopeExits.json

```diff
@@ -1,14 +1,14 @@
 {
     "openapi":"3.0.0",
     "info":{
-        "title":"TestConfiguration",
-        "description":"OpenApi document auto-generated by Frank!Framework for TestConfiguration",
-        "version":"Not specified"
+        "title":"TestConfiguration (xxx)",
```

Comment:
De stage indicatie is nu verhuisd van server naar title, terwijl het toch eigenlijk alleen maar de server is die de stage bepaalt.